scss/base/_utilities.scss

```diff
@@ -26,6 +26,42 @@
   @include text-truncate;
 }
 
+.oo-absolute-centered {
+  @include absolute-center;
+}
+
+.oo-flex-row-parent {
+  @include flex-center-children(row);
+  white-space: nowrap;
+  flex-flow: row nowrap;
+}
+
+.oo-flex-row {
+  @include flexbox;
+  @include flex-direction(row);
+  flex: 1;
+}
+
+.oo-flex-row:first-child > * { margin-right: auto; }
+
+.oo-flex-row:last-child > * { margin-left: auto; }
+
+.oo-flex-column-parent {
+  @include flex-center-children(column);
+  white-space: nowrap;
+  flex-flow: column nowrap;
+}
+
+.oo-flex-column {
+  @include flexbox;
+  @include flex-direction(column);
+  flex: 1;
+}
+
+.oo-flex-column:first-child > * { margin-bottom: auto; }
+
+.oo-flex-column:last-child > * { margin-top: auto; }
```

Comment:
Same as previous comment, I think it would better to avoid the `*` selector if possible